src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -480,8 +481,8 @@ namespace Akka.Actor
     public class DeadLetterActorRef : Akka.Actor.EmptyLocalActorRef
     {
         public DeadLetterActorRef(Akka.Actor.IActorRefProvider provider, Akka.Actor.ActorPath path, Akka.Event.EventStream eventStream) { }
-        protected override void HandleDeadLetter(Akka.Event.DeadLetter deadLetter) { }
         protected override bool SpecialHandle(object message, Akka.Actor.IActorRef sender) { }
+        protected override void TellInternal(object message, Akka.Actor.IActorRef sender) { }
```

Comment:
Consolidated the `EmptyActorRef` and the `DeadLetterActorRef` to look more like the JVM. API modifications consisted of `protected` methods only.